1/Dockerfile.rhel7

```diff
@@ -46,7 +46,9 @@ COPY ./contrib/openshift /opt/openshift
 COPY ./contrib/jenkins /usr/local/bin
 ADD ./contrib/s2i /usr/libexec/s2i
 
-RUN /usr/local/bin/fix-permissions /opt/openshift && \
+RUN mkdir -p /tmp/openshift/plugins && \
+    ln -s /usr/lib64/jenkins/openshift-pipeline.hpi /opt/openshift/plugins/openshift-pipeline.hpi && \
```

Comment:
Well ... i just noticed that I had a typo ... /tmp/openshift/plugins vs. /opt/openshift/plugins ...the plugins subdir did not exist in /opt/openshift when I exec/ran the rhel7 container earlier, and on the centos side, the plugins.sh script that is runn is doing the mkdir -p /opt/openshift/plugins.

I'll try fixing the typo first, and then if it still complains, i'll move the `mkdir` and `ln` into a simple script analogous to the one ran in the centos image.